### DISCUSSION TITLE
Is this fine? (JLCPCB BOM question)

### DISCUSSION THREAD

**Discussion by usagi-epta** [adamb314/ServoProject#28] [Q&A] [answered]
![IMG_9409](https://github.com/adamb314/ServoProject/assets/763503/f1324ce6-d775-4af3-b07c-d00f52dd82ab)
![IMG_9410](https://github.com/adamb314/ServoProject/assets/763503/eb91bb69-d8a7-42b2-9c9e-96dd3429f651)
I’ve only ever been a novice tinkerer and am only just dipping my toes back into making stuff after years of not touching electronics (majorly to play with this project), and it’s my first time ordering PCBs from JLCPCB so I am unsure if the above is fine.

**Comment by Cydget** [comment, marked as answer]
Yes, its the current limiting resistor for the status LED on the board. I'm guessing the change was made to commonize components. Only difference between using a 2.2k ohm and 8.2kohm resistor for R6 is that a 2.2ohm will make the LED much brighter(not always a desirable effect) . I recommend ordering as is with the duplicate part #'s

**Reply by usagi-epta** [reply]
Thank you, I know just enough to get me into trouble 😅
